packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md

```diff
@@ -9,38 +9,39 @@ Use text input to gather free-form text from a user.
 
 ## Elements
 
-![Elements](./img/input-elements3.png)
+<img src="./img/textinput-elements.png" alt="Text input elements" width="488"/>
 
-1. Label
-2. Input field
-3. [Help](#help) - provides contextual assistance like popovers, placeholder text, or helper text, in any area where the user might need additional information to provide the right input.
-4. [Errors and validation](#errors-and-validation) - ensures that the correct information is collected before the form can be submitted to the system.
+1. **Label:** Indicates what you want the user to input in the text field.
+2. **Input field:** Area for user to type in their text input.
+3. **[Help](#help):** Provides contextual assistance like popovers, placeholder text, or helper text, in any area where the user might need additional information to provide the right input.
+4. **[Errors and validation](#errors-and-validation):** Ensures that the correct information is collected before the form can be submitted to the system.
 
 ## Usage
 * Be sure to size input fields appropriately to anticipate the longest likely value (if a maximum character length is known).
 * When a value is too long to be fully displayed in a text field, use [truncation](/design-guidelines/content/grammar-and-terminology#truncation).
 * Provide a [text area](/documentation/react/components/textarea) for input anticipated to be longer than a single line.
-
-![Text area](./img/textarea.png)
-
-* Pre-populate fields with known values when possible, e.g. a default IP address.
+* Pre-populate fields with known values when possible, like a default IP address, for example.
 * When presented in a form, the initial input field is usually the first input field required. The primary input field should receive focus when the form is presented to the user.
 
 ### Help
```

Comment:
This page talks further about how and when to use different forms of on-screen help: https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/embedded-assistance  We should reference and link this.